packages/dashboard/src/app/views/shared/bodyViewOptions.js

```diff
@@ -119,6 +128,10 @@ export default function BodyViewOptions({
                     noMatchesFoundLabel={filter.noMatchesFoundLabel}
                     searchPlaceholder={filter.searchPlaceholder}
                     offsetOverride
+                    maxWidth={FILTER_MAX_WIDTH}
```

Comment:
May be too late to bring this up, but do this list items require a max-width to not look funky? It's acceptable behavior for custom-styled drop-downs - I've seen both styles in the wild.

Reply:
The `maxWidth` prop here is technically only used to style to `Popup` components width.  There is some logic added to the `Popup` component that now depends on this prop to keep these popups from being absurdly to big (ie if the user wants an 'option' thats a 200 word paragraph).  This way any where that uses the 'datalist' can now specify a `maxWidth` for the popup.  But if you have a suggestion on how make it look cleaner I can defiantly look into it.